Support/Multitenancy/Multiple-Ironic-conductors/build-sushy-tools-image.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+IMAGE_NAME="127.0.0.1:5000/localimages/sushy-tools"
+SUSHYTOOLS_DIR="/tmp/sushy-tools"
+
+if [[ ${1:-""} == "-f" ]]; then
+  rm -rf "${SUSHYTOOLS_DIR}"
+  docker rmi "${IMAGE_NAME}"
+fi
+
+if [[ $(docker images | grep ${IMAGE_NAME}) != "" ]]; then
```

Comment:
```suggestion
if docker images | grep -q "${IMAGE_NAME}"; then
```
Let's just use return values.